src/components/Table/SimpleTable.js

```diff
@@ -47,11 +48,14 @@ class SimpleTable extends React.Component {
   };
 
   static defaultProps = {
-    renderRow: (item) => (
+    renderRow: item =>
       <Table.Row key={JSON.stringify(item)}>
-        {Object.values(item).map((val) => <Table.Cell>{defaultValueRenderer(val)}</Table.Cell>)}
-      </Table.Row>
-    ),
+        {Object.values(item).map(val =>
```

Comment:
FYI: Lodash checks used above but es6 was used for this check, could use `_.values` for consistency (not really a big issue, just a heads up)